cmd/remoteva/main.go

```diff
@@ -65,6 +84,10 @@ func main() {
 	tlsConfig, err := c.RVA.TLS.Load(scope)
 	cmd.FailOnError(err, "tlsConfig config")
 
+	if c.RVA.ServerUseAlternateClientAuthPolicy {
+		tlsConfig.ClientAuth = tls.VerifyClientCertIfGiven
+	}
```

Comment:
Are there any other checks we can do before enabling this? None immediately spring to my mind. However, I'm going to give this a little more thought as well. Ideally this would fail out on start if it wasn't running in the expected environment.

Reply:
Expected environment meaning staging or production?

Reply:
We discussed this offline and I am going to punt this down the road for a future PR.

Reply:
That's not what I was getting at but it does give me an idea. What I meant was, "Can we detect that we're running outside of our own DCs before we okay this flag being set.". However, you've also just given me the idea that perhaps we can detect that we _are_ running on-premise and fail out if this flag is set.

I don't want to hold up this PR, if the ask would be too complicated feel free to say so.

Reply:
That is a question for future Phil. He's a ch~u~amp!